examples/widgets/spinbox/index.rst

```diff
@@ -1,5 +1,5 @@
 
-Simple Spinbox
+Simple Spinbox with cursor style added
```

Comment:
The length of `------` under the title should match the length of the title.

Reply:
Fixed in next patch.